gitcloner

```diff
@@ -32,5 +32,4 @@ def main():
     account.clone_repos()
 
 
-if __name__ == '__main__':
-    main()
+main()
```

Comment:
@shakib609 why did you remove that wrapper? this makes sure that the code is not executed when included in another script.